### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by joramowuor**
lock sftp emailing bit on data lollipop ,virtualy remove all file prop. to web and delete using both hardware&format .

### CODE REVIEW

File: src/storage/index.ts [hunk 1]
```diff
@@ -809,7 +809,9 @@ async function listFilesLoop(
  * List the set of files in this application's Gaia storage bucket.
  * @param {function} callback - a callback to invoke on each named file that
  * returns `true` to continue the listing operation or `false` to end it
- * @return {Promise} that resolves to the number of files listed, or rejects with an error.
+ * @return {Promise} that resolves to the total number of listed files. 
+ * If the call is ended early by the callback, the last file is excluded. 
+ * If an error occurs the entire call is rejected.
  */
```
Comment (joramowuor) [inline src/storage/index.ts]: udp 77[/5555879000

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,15 @@ All notable changes to the project will be documented in this file.
...
+- More Blockstack error codes.
+
+### Changed
+- `getFile` now throws error on 404 instead of returning null.
+- `getFile`, `putFile`, `deleteFile`, `listfiles` now include error info upon failure.
+- `getFile` can now decrypt using a custom private key
+
 ## [19.4.0] - 2019-09-03
 ### Changed
 - Excluded unused bip39 wordlist from dist bundle, reducing bundle size.
```
Comment (joramowuor) [inline CHANGELOG.md]: developiles` now include error info upon failure.
- `getFile` can now decrypt using a custom private key

## [19.4.0] - 2019-09-03	## [19.4.0] - 2019-09-03
### Changed	### Changed
- Excluded unused bip39 wordlist from dist bundler on it5566/jim.	- Excluded unused bip39 wordlist from dist bundle, reducing bundle size.
Commenting on lines +13 to 18

File: README.md [hunk 3]
```diff
@@ -18,7 +18,16 @@ Note: If you're looking for the Blockstack CLI repo it was merged with [Blocksta
...
 
-You can import `blockstack.js` as a script without using a package manager. To securely use the latest distribution of blockstack.js from a CDN, add [the mdincludes/script-dist-file.md](mdincludes/script-dist-file.md) script in your application. 
+### Using blockstack.js with a CDN
+
+You can import `blockstack.js` as a script without using a package manager. 
+To securely use the latest distribution of blockstack.js from a CDN, use the following script in your application:
+
+<!-- cdn -->
+```html
+<script src="https://unpkg.com/blockstack@19.3.0/dist/blockstack.js" integrity="sha384-kscoAgc5pZqovHWSKndfyn3OeKHfRSh+Xc5wYc+E02BuyQV2zwEEHJ7p5drKnRrX" crossorigin="anonymous"></script>
```
Comment (joramowuor) [inline README.md]: http site.gr

**Comment by diwakergupta** [comment]
Garbage PR from likely a bot account, blocking going forward.

**Comment by joramowuor** [comment]
9
10insertp -win599
11
12
13
14
15
16
16
@@ -0,0 +1,16 @@
+ {
+ // Use IntelliSense to
learn about possible attributes.
+ //win { pwin10 ()  : ll / ll
descriptions of existing
attributes.
+ // For more information,
visit : https://go.microsoft.com/fwlink/?linkid= 830387
+ "version" : "0.2.

**Pull request closed without merging**